artifactory/commands/transferconfig/transferconfigmerge_test.go

```diff
@@ -0,0 +1,96 @@
+package transferconfig
+
+import (
+	"github.com/jfrog/jfrog-client-go/access/services"
+	"github.com/stretchr/testify/assert"
+	"os"
+	"strings"
+	"testing"
+)
+
+const (
+	QuotaNumber = 1073741825
+)
+
+func TestCreateAndValidateConflicts(t *testing.T) {
+	// todo: remove!
+	gitenv := os.Getenv("GITHUB_ENV")
```

Comment:
We should use the same standard way for creating temp files, as we're doing for other tests. We should also delete them inside a defer.